tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/create_expected_ct_list_test.py

```diff
@@ -168,8 +179,15 @@ def test_store_expected_ct_list(self):
                 ],  # verifying the correct fields and data are sandboxed here
                 'fq_sandbox_table_name':
                     self.fq_sandbox_table_names[0],
-                'sandbox_fields': ['research_id'],
-                'sandboxed_ids': [80, 60, 50, 40, 20]
+                'sandbox_fields': ['research_id', 'participant_id', 'is_aian'],
+                'sandboxed_ids': [80, 60, 50, 40, 20],
+                'sandboxed_values': [
+                    (80, 800, 'no'),
+                    (60, 600, 'no'),
+                    (50, 500, 'yes'),
+                    (40, 400, 'no'),
+                    (20, 200, 'no'),
```

Comment:
This is not actually checking the values list or used by the base class yet.  You need to write that functionality into the base class.  It will be similar to the way `cleaned_values` are checked.  See [lines 319-321](https://github.com/all-of-us/curation/blob/develop/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py#L319 ) for an example of how to do this.  Implement the change in the range of lines 330-335.